hw/riscv/virt.c

```diff
@@ -265,15 +296,27 @@ static void riscv_virt_board_init(MachineState *machine)
         main_mem);
 
     /* create device tree */
-    create_fdt(s, memmap, machine->ram_size, machine->kernel_cmdline);
+    fdt = create_fdt(s, memmap, machine->ram_size,
```

Comment:
we should avoid whitespace changes to code that isn't changing

Reply:
Fixed